net-misc/curl/curl-9999.ebuild

```diff
@@ -22,10 +22,10 @@ fi
 
 LICENSE="BSD curl ISC test? ( BSD-4 )"
 SLOT="0"
-IUSE="+adns alt-svc brotli +ftp gnutls gopher hsts +http2 idn +imap kerberos ldap mbedtls +openssl +pop3 +progress-meter rtmp rustls samba +smtp ssh ssl sslv3 static-libs test telnet +tftp websockets zstd"
+IUSE="+adns +alt-svc brotli +ftp gnutls gopher +hsts +http2 idn +imap kerberos ldap mbedtls nghttp3 +openssl +pop3"
```

Comment:
Why are we enabling alt-svc and hsts by default? And why are they USE flags to begin with?

Reply:
I can't see any good reason in 2023 _not_ to have HSTS and alt-svc on by default.

Why are they USE flags to begin with? Good question; they don't change any dependencies. cURL has historically had mountains of USE flags - it's a holdover from that - I considered dropping the USE for both (and probably will in a later version).